content/sensu-go/5.6/api/overview.md

```diff
@@ -120,6 +121,79 @@ Content-Type: application/json
 ]
 {{< /highlight >}}
 
+### Filtering
+
+**ENTERPRISE ONLY**: API filtering in Sensu Go requires a Sensu Enterprise license. To activate your Sensu Enterprise license, see the [getting started guide][7].
+
+The Sensu API supports filtering for all GET endpoints that return an array. You can filter resources based on their labels with a label selector using the `labelSelector` query parameter and on certain pre-determined fields with a field selector using the `fieldSelector` query parameter.
+
+For example, the following request filters the response to only include resources that have a label entry `region` with the value `us-west-1`. We will use the flag `--data-urlencode` in curl so it encodes the query parameter for us, in conjunction with the `-G` flag to it appends the data to the URL.
```

Comment:
At the end of the line, should it read "flag so it appends the data" instead of "flag to it appends the data"?